os/kernel/Kconfig

```diff
@@ -727,39 +727,12 @@ config SCHED_ATEXIT
 	---help---
 		Enables the atexit() API
 
-config SCHED_ATEXIT_MAX
```

Comment:
In POSIX description of atexit, below is written.
```
The application should call sysconf() to obtain the value of {ATEXIT_MAX},
the number of functions that can be registered.
There is no way for an application to tell
 how many functions have already been registered with atexit().
```

Do we need to remove this?

Reply:
1. I dont think we have sysconf() support.
2. SCHED_ATEXIT_MAX  was used for allocating fixed size atexit callback function array,  since we planning to use linked list, we dont need it anymore.
Do you see any problem with this approach?

Reply:
Just FYI .
This fix is related to media_utc memory leak and repeated execution failure.

1. Memory Leak:
We have two sub issues,
1. Leak in libc++
2. Leak due to inode_alloc. This would help to free the memory allocated by inode.

2. Repeated execution of media_utc doesn’t work:
It is due to static object issue. If you run media_utc for the second time, it won’t work because static objects are not getting constructed for the second and onward case.

Detailed history can be found at WCTIAPP-2707